Forge/src/main/java/me/ferdz/placeableitems/network/handler/AnonymousPacketHandler.java

```diff
@@ -0,0 +1,24 @@
+package me.ferdz.placeableitems.network.handler;
+
+import java.util.function.Supplier;
+
+import me.ferdz.placeableitems.network.SUpdateFluidHolderPacket;
+
+import net.minecraftforge.fml.network.NetworkEvent;
+
+/**
+ * Represents an anonymous packet handler. 
+ *
+ * @author Parker Hawke - Choco
+ */
+public interface AnonymousPacketHandler {
+
+    /**
+     * Handle the {@link SUpdateFluidHolderPacket}.
+     *
+     * @param packet the packet to handle
+     * @param ctx the network context
+     */
+    public void handleUpdateFluidHolder(SUpdateFluidHolderPacket packet, Supplier<NetworkEvent.Context> ctx);
```

Comment:
I like that this is expandable, good job

Reply:
That was the goal was to have it be easy to add new handlers :)